nosekatreport/test_aqf.py

```diff
@@ -25,11 +25,15 @@ def setUp(self):
     def tearDown(self):
         pass
 
-    @aqf_vr('VR.EXAMPLE.FC.4', 'VR.EXAMPLE.FC.6', 'VR.EXAMPLE.FC.9')
+    @aqf_vr('VR.EXAMPLE.FC.4', 'VR.EXAMPLE.FC.6', 'VR.EXAMPLE.FC.9', 'CBF-REQ-0126', 'CBF-REQ-0047')
+    @aqf_vr('TP.C.1.19')
```

Comment:
It seems to work with two of the same decorators on the method, but why not just extend the list of arguments for the first one?  It could wrap over multiple lines.

Reply:
Come to think of it, maybe I should but then.
Flake8 won't like it:  E501 line too long

Reply:
You can wrap it over multiple lines.  Is flake8 OK with this:
```
@aqf_vr('VR.EXAMPLE.FC.4', 'VR.EXAMPLE.FC.6', 'VR.EXAMPLE.FC.9', 'CBF-REQ-0126',
        'CBF-REQ-0047', 'TP.C.1.19')
```

Reply:
Yes, should be happy.